### PR TITLE
Rewriting and reorganization of front matter

### PR DESCRIPTION
**Pull request by markkramerus**

### CODE REVIEW

File: spec/IndexFolder_Oncocore/index.html [hunk 1]
```diff
@@ -8,48 +8,210 @@ <h1>Contents of this Implementation Guide</h1>
...
+    <ul>
+        <li><a href="https://cancerstaging.org/references-tools/deskreferences/Pages/Breast-Cancer-Staging.aspx" target="_blank">American Joint Committee on Cancer (AJCC) Staging Manual (8th Edition) Breast Cancer Chapter</a></li>
+        <li><a href="https://www.cap.org/protocols-and-guidelines" target="_blank">College of American Pathologists (CAP) Cancer Protocols</a></li>
+        <li><a href="https://www.naaccr.org/SSDI/SSDI-Manual.pdf?v=1531675132" target="_blank">North American Association of Central Cancer Registries (NAACCR) 2018 Site-Specific Data Items Manual</a></li>
+        <li><a href="http://www.hl7.org/implement/standards/product_brief.cfm?product_id=383" target="_blank">HL7 CDA R2 IG: Reporting to Public Health Cancer Registries from Ambulatory Healthcare Providers</a></li>
+        <li><a href="https://www.nccn.org/professionals/physician_gls/default.aspx#site" target="_blank">National Comprehensive Cancer Network (NCCN) Clinical Practice Guidelines in Oncology</a></li>
+        <li><a href="http://recist.eortc.org/wp-content/uploads/2015/03/RECISTGuidelines.pdf" target="_blank">RECIST Guidelines v1.1</a></li>
```
Comment (mlterryMitre) [inline spec/IndexFolder_Oncocore/index.html]: URL is broken.  Replace with https://project.eortc.org/recist/wp-content/uploads/sites/4/2015/03/RECISTGuidelines.pdf.

File: spec/IndexFolder_Oncocore/index.html [hunk 2]
```diff
@@ -8,48 +8,210 @@ <h1>Contents of this Implementation Guide</h1>
...
+        <li><a href="https://www.hl7.org/fhir/genomics.html" target="_blank">HL7 FHIR Genomics Implementation Guidance</a></li>
+        <li><a href="https://www.ncbi.nlm.nih.gov/gtr" target="_blank">The National Center for Biotechnology Genetic Test Registry (GTR)</a></li>
+        <li><a href="http://www.hgvs.org/" target="_blank">The Human Genome Variation Society nomenclature for defining genetic variants</a>.</li>
```
Comment (mlterryMitre) [inline spec/IndexFolder_Oncocore/index.html]: www.hgvs.org is the society.  Replace with https://varnomen.hgvs.org/ which is specific to the nomenclature defined by HGVS.

File: spec/IndexFolder_Oncocore/index.html (hunk 1 above)
Comment (markkramerus) [reply spec/IndexFolder_Oncocore/index.html]: fixed.

File: spec/IndexFolder_Oncocore/index.html (hunk 2 above)
Comment (markkramerus) [reply spec/IndexFolder_Oncocore/index.html]: fixed.

**Pull request merged**